src/scripts/main.js

```diff
@@ -357,4 +357,21 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the DOM before attempting to select it. If it doesn't exist, `document.querySelector('.dashboard')` will return `null`, and attempting to append children to `null` will cause an error. Consider adding a check to ensure `table` is not `null` before proceeding.